rebar.config

```diff
@@ -6,6 +6,7 @@
     {riak_dt, ".*", {git, "git://github.com/aletomsic/riak_dt", {tag, "type_check_antidote"}}},
     {riak_api, ".*", {git, "git://github.com/basho/riak_api", {tag, "2.0.2"}}},
     {riak_pb, ".*", {git, "git://github.com/syncfree/riak_pb", {tag, "new_interface"}}},
+    {erlzmq, ".", {git, "git://github.com/tcrain/erlzmq2", "master"}},
```

Comment:
The only change introduced by Tyler is the following one: https://github.com/tcrain/erlzmq2/commit/95bb40064d3d7b252c57987f23fd67a870028db1

This was done because the default tarball address was unreachable from grid5k. I do not know if the issue still persists - we've had lots of problems with blocked traffic on grid5k.